src/Service.Host/client/src/itemTypes.js

```diff
@@ -120,3 +120,5 @@ export const vendorManagers = new ItemType(
 );
 
 export const planners = new ItemType('planners', 'PLANNERS', '/purchasing/suppliers/planners');
+
+export const contacts = new ItemType('contacts', 'CONTACTS', '/purchasing/suppliers/contacts');
```

Comment:
i would rename this supplierContacts whereever you can because contacts are still used in a lot of places and this could potentially become confusing

Reply:
I think this can go actually. It was a hangover from when I was allowing a lookup on existing contacts